### PR TITLE
step out of wrapper calls

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: src/commands.jl [hunk 1]
```diff
@@ -339,8 +343,14 @@ function maybe_reset_frame!(@nospecialize(recurse), frame::Frame, @nospecialize(
         frame = caller(frame)
         frame === nothing && return nothing
         frame.callee = nothing
+        ssavals = frame.framedata.ssavalues
+        is_wrapper = isassigned(ssavals, frame.pc) && coalesce(ssavals[frame.pc] == Wrapper(), false)
```
Comment (KristofferC) [inline src/commands.jl]: This `coalesce` was fun...

File: test/debug.jl [hunk 2]
```diff
@@ -154,7 +154,7 @@ struct B{T} end
         frame = JuliaInterpreter.enter_call(f, 2; b = 4)
         fr = JuliaInterpreter.maybe_step_through_wrapper!(frame)
         fr, pc = debug_command(fr, :nc)
-        fr, pc = debug_command(fr, :nc)
```
Comment (KristofferC) [inline test/debug.jl]: This now actually finishes the frame.

File: src/commands.jl (hunk 1 above)
Comment (KristofferC) [reply src/commands.jl]: Actually, can just use `===`

**Comment by timholy** [comment]
Beautiful!

**Pull request merged**